### ISSUE TITLE
Freeform mode is ugly

### ISSUE THREAD

**Issue by pyamsoft** [pyamsoft/padlock#17]
In free form mode, PadLock will sometimes launch the LockScreen, and sometimes not

When it does, it will be rather small, and will not obscure the view of the activity. This being said, as long as it responds to free form events, it will restrict input to an application by grabbing input whenever the app is focused.

Due to the way padlock works currently (see LockScreenActivity and PadLockService) it will launch a new LockScreen each time a new freeform window, or the same freeform window, comes back into focus. This can lead to multiple freeform windows for the same application.

In multiwindow mode, this is what we want so that we prevent the user from opening an app, putting the device in multi window mode, and then opening the same app in the other multi window.

TL:DR Free form is ugly, and buggy, and half works. If the goal is to restrict input then the current solution seems to work "okay" but if our goal is to restrict entire application access (it is) then freeform mode causes the app to fail.
